### DISCUSSION TITLE
Hono passport integration

### DISCUSSION THREAD

**Discussion by yehtetaung4466** [honojs/hono#2004] [Q&A]
How do i integrate passport strategies to hono for auth?Does anyone have example apps?

**Comment by ghost** [comment]
Hono has built-in auth middleware.
[This is an example of using it](https://github.com/honojs/examples/tree/f5996d36d29cfced90d514bf1cdb62eef9f014eb/blog).

And since passport allows us to invoke all the actions involved in authentication, we can make it our own.
express but passport can refer to [this code](https://github.com/passport/todos-express-google/blob/main/routes/auth.js).

Hope it works.
